docs/atomic-cluster.1.md

```diff
@@ -0,0 +1,65 @@
+% ATOMIC(1) Atomic Man Pages
+% Matthew Barnes
+% April 2016
+# NAME
+atomic-cluster - Manage cluster hosts
+
+# SYNOPSIS
+**atomic cluster [OPTIONS] SUBCOMMAND**
+
+**atomic cluster** will issue commands to a Commissaire server over HTTP.
+
+Commissaire allows administrators of a Kubernetes, Atomic Enterprise or
+OpenShift installation to perform administrative tasks without the need
+to write custom scripts or manually intervene on systems.
+
+# NOTE
+The `cluster` subcommand is only available when the Commissaire client
+module `commctl` is installed.
+
+# OPTIONS
+**-h** **-help**
+  Print usage statement.
+
+# SUBCOMMANDS
+**get cluster** CLUSTER_NAME
+  Report status of the specified cluster.
```

Comment:
Perhaps we should setup a bluejeans to discuss this.  I really do not understand what Commissaire does, so tough for me to give opinion.